savoirfairelinux/sous-chef
# fix368a delivery route sheet initial version

## Fixes #368 partially : will add functionalities, refactor, clean up code, comment, add tests.

### Changes proposed in this pull request:

* In Organize route map page, Save button now saves (in Route model) the chosen sequence for the weekday for this route
* In Organize route map page, saved sequence from last week will be remembered for matching clients
* Route Sheet Page uses sequence from visual route map
* Added 5th step = "Route Sheet" to Kitchen Count (but see comment in Issue #368 for an alternative)
* New fixture containing test data for delivery app

### Status


### How to verify this change

* Click Kitchen Count
* Generate orders for the day
* Click OK I'm ready
* Choose Ginger Pork Main dish
* Click Next to see kitchen count report
* Click Next Organize routes
* Click pull-down to change route 
* Click pull-down to change route, selecting a route having clients with orders today
* Play with sequence
* Click "Save"
* will see popup that says saved, click OK
* Click Next Delivery Route Sheet : will see Route Sheet ordered using saved sequence. Note : The "special" remark on main dishes for clients with restrictions, and their counts at the top are not yet displayed; I need to discuss this requirement with JF since we will have labels on each dish)

### Deployment notes and migration

- [X] Migration is needed for this change (added field in member:Route model)
